modules/fscloud/README.md

```diff
@@ -0,0 +1,70 @@
+# Profile for IBM Cloud Framework for Financial Services
+
+This code is a version of the [parent root module](../../) that includes a default configuration that complies with the relevant controls from the [IBM Cloud Framework for Financial Services](https://cloud.ibm.com/docs/framework-financial-services?topic=framework-financial-services-about). See the [Example for IBM Cloud Framework for Financial Services](/examples/fscloud/) for logic that uses this module. The profile assumes you are deploying into an account that is in compliance with the framework.
```

Comment:
you will need to update the parent root module link to a full link for it to work in hashicorp registry

Reply:
I guess its just a repo URL in case we go ahead with the full link for the parent root module.